src/test/java/org/folio/search/controller/IndexControllerTest.java

```diff
@@ -52,6 +58,102 @@ void createIndex_positive() throws Exception {
       .andExpect(jsonPath("$.indices", is(List.of(INDEX_NAME))));
   }
 
+  @Test
+  void createIndex_negative_indexAlreadyExists() throws Exception {
+    var requestBuilder = post("/search/index/indices")
+      .content(asJsonString(requestBody()))
+      .contentType(APPLICATION_JSON)
+      .header(X_OKAPI_TENANT_HEADER, TENANT_ID);
+
+    var elasticsearchException = new ElasticsearchException("Elasticsearch exception "
+      + "[type=resource_already_exists_exception, "
+      + "reason=index [instance_test-tenant/um_SBtCaRLKUOBbdmFZeKQ] already exists]");
+    elasticsearchException.setIndex(new Index(INDEX_NAME, randomId()));
+
+    when(indexService.createIndex(RESOURCE_NAME, TENANT_ID)).thenThrow(
+      new SearchOperationException("error", elasticsearchException));
+
+    mockMvc.perform(requestBuilder)
+      .andExpect(status().isBadRequest())
+      .andExpect(jsonPath("$.errors[0].message", is("Index already exists: " + INDEX_NAME)))
+      .andExpect(jsonPath("$.errors[0].type", is("ElasticsearchException")))
+      .andExpect(jsonPath("$.errors[0].code", is("Elasticsearch error")))
+      .andExpect(jsonPath("$.total_records", is(1)));
```

Comment:
Can it be the first assertion?

Reply:
why? any of this assertion will fail if response object does not contain correct value at the json path value, what order of assertions can change test behaviour?

Reply:
It should be a little bit clearer to find out cause if test is failed. 

If `errors` is empty, I assume the assertion will fail with something like: expected `Index already exists: instance` but actual `null` and it is not clear whether there were no message or errors is empty.